app/src/main/java/com/courseproject/tindar/usecases/chat/ChatPresenter.java

```diff
@@ -0,0 +1,28 @@
+package com.courseproject.tindar.usecases.chat;
+
+import com.courseproject.tindar.entities.MessageModel;
+
+import java.util.ArrayList;
+
+/**
+ * Creates and maintains an ArrayList of MessageModels for ChatActivity to display.
+ */
+public interface ChatPresenter {
```

Comment:
This is not presenter. This is more ChatInputBoundary - interface for the interactor